src/Sylius/Component/Core/Model/ShopUser.php

```diff
@@ -22,6 +22,11 @@ class ShopUser extends BaseUser implements ShopUserInterface
     /** @var BaseCustomerInterface|null */
     protected $customer;
 
+    public function isLocked(): bool
+    {
+        return $this->locked;
+    }
+
```

Comment:
```suggestion
```
Should be reverted, there's already a method that does the same, only the other way around `isAccountNonLocked`